trytond/sendmail.py

```diff
@@ -97,8 +99,7 @@ def get_smtp_server(uri=None, strict=False):
         server = connector(uri.hostname, uri.port, **extra)
     except Exception:
         if strict:
-            raise
-        logger.error('fail to connect to %s', uri, exc_info=True)
+            raise ValidationError(gettext('ir.msg_missing_configuration'))
```

Comment:
ce serait bien de garder une ligne logger.error('fail to connect to %s', uri, exc_info=True) dans le if, pour avoir une trace de l'exception originale dans les logs du serveur.